mappings/net/minecraft/client/util/math/Vector3f.mapping

```diff
@@ -1,4 +1,7 @@
 CLASS net/minecraft/class_1160 net/minecraft/client/util/math/Vector3f
+	FIELD field_20703 pitch Lnet/minecraft/class_1160;
+	FIELD field_20705 yaw Lnet/minecraft/class_1160;
+	FIELD field_20707 roll Lnet/minecraft/class_1160;
```

Comment:
this is like a positive X/Y/Z unit vector instead of pitch/yaw/roll. Note that this field should never be final because this Vector3f class is actually mutable. Also please name the negative x/y/z unit vectors.